source/scripting_v3/GTA.NaturalMotion/EuphoriaBase.cs

```diff
@@ -214,7 +319,34 @@ public void Start()
 		public void Start(int duration)
 		{
 			_message.SendTo(_ped, duration);
-			_message.ResetArguments();
+		}
+		/// <summary>
+		/// Starts this NaturalMotion behavior on the <see cref="Ped"/> from the beginning without starting a scripted NM ragdoll task.
+		/// </summary>
+		/// <param name="ifNMScriptControlRunning">
+		/// <para>Specifies whether the messages should be sent to the only if they <see cref="Ped"/> has a <c>CTaskNMScriptControl</c> running, which can be started with <see cref="Ped.Ragdoll(int, RagdollType)"/>.</para>
+		/// <para>
+		/// If you set this parameter to <see langword="false"/> and you need to set the <see cref="Ped"/> from a ragdoll state to an animated state after a certain timeout (in game time),
+		/// you should call <see cref="Ped.CancelRagdoll()"/> on them after a certain timeout if they are alive and <see cref="Ped.IsRagdoll"/> returns <see langword="true"/>.
+		/// </para>
+		/// </param>
+		public void StartNoNewRagdollTask(bool ifNMScriptControlRunning = true)
```

Comment:
As above, SHVDN should automatically detect wether a new ragdoll task is needed or not, rather than having two separate functions.